### DISCUSSION TITLE
gnb 通过配置文件启动失败

### DISCUSSION THREAD

**Discussion by meg4152** [opengnb/opengnb#17] [Q&A]
如图, `gnb -c ./gnb/conf/2001` 失败, `ip addr`没有显示对应设备, `netstat -an | grep 23402`亦无输出.
对应的配置文件也如图.
![Screenshot from 2022-06-08 19-29-56](https://user-images.githubusercontent.com/40293541/172614039-d50358a1-a34f-4652-a89a-77c199cf5280.png)
![Screenshot from 2022-06-08 20-26-04](https://user-images.githubusercontent.com/40293541/172615574-708b8e3d-0df7-4158-9be6-004c4f35f6f0.png)
其中gnb.map为自动生成.
`gnb_ctl -b ./gnb.map -c` 输出 open ctl block error [./gnb.map]

**Comment by gnbdev** [comment]
QQ微信同号 26253
>

**Reply by harvey-git** [reply]
从图上看，你这是群晖之类的阉割linux吧，缺少一些必要的组件比如tun相关的模块！令我回忆起当年在群晖上配置screen时解决相关依赖的恐惧！

**Reply by gnbdev** [reply]
我忘记这位用户后来有没加我微信解决问题，不太明白你在图中什么地方可以看出他用的是经过剪裁的没有tun内核模块的Linux，没给出gnb的日志，又没给出 strace 的信息，这个情况基本无法判断。
